src/gsi/cplr_read_wrf_mass_guess.f90

```diff
@@ -1418,7 +1447,11 @@ subroutine read_wrf_mass_netcdf_guess_wrf(this,mype)
                        indx_seas1, indx_seas2, indx_seas3, indx_seas4,indx_p25
     character(len=5),allocatable :: cvar(:)
     real(r_kind)   :: ges_rho, tsn
-  
+
+  ! Logarithmic q option related variables
+    real(r_kind) :: qr_min, qs_min, qg_min
+    real(r_kind) :: qr_thrshd, qs_thrshd, qg_thrshd
```

Comment:
These six variables, qr_min, qs_min, qg_min, qr_thrshd, qs_thrshd, and qg_thrshd, have been identified as unused.  Please remove unused variables.

Reply:
We deleted them as you suggested.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-9454084e44d25b1106008105c0d11d4efee5640bbe7134a7f8faa41509269918R1450